config/settings.yml

```diff
@@ -135,7 +125,7 @@ split_accounts:
     - administrator
     # - account_manager
 
-# This may be overridden in settings.local.yml if your fork is using S3, so
+# This may be overridden in secrets.yml if your fork is using S3, so
```

Comment:
This part isn't quite true as it stands. I think it might be nice to think about how this can work. 

What might work is to do a `Settings.paperclip.merge(Rails.application.secrets.paperclip)` and then we can get the best of both worlds. Thoughts on that?

Reply:
Hmm. Where would we actually do the merge?

Reply:
Looks like we use the paperclip settings in `app/models/concerns/downloadable_file.rb`. So probably in there.